platform-sdk/sdk/settings.txt

```diff
@@ -46,3 +46,5 @@ prometheus.endpointEnabled,               false
                                           #############################
 
 loadKeysFromPfxFiles,                     false
+crypto.cpuVerifierThreadRatio,            0.1

```

Comment:
Yes.

As a part of this PR, when running a multiple nodes in the same JVM, the platform context for each node now has a different cryptography instance. Previously it would always share the same cyrptography instance between all nodes in the same JVM. 

Since laptop resources are in high demand, I figured if we are having 4x the number of cryptography instances, it makes sense to reduce the size of the thread pool for cryptography.